### DISCUSSION TITLE
docker-slim command not found

### DISCUSSION THREAD

**Discussion by dheeruymv** [slimtoolkit/slim#355] [General]
Hi All,

I've started using docker-slim.

My machine:
NAME="Red Hat Enterprise Linux"
VERSION="8.3 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.3"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.3 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8.3:GA"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.3
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.3"

with root user:

- Untar of ds.tar.gz
- Move of docker-slim and docker-slim-sensor to /usr/local/bin

when i run docker-slim command, it says docker-slim : command not found.

PS: gave 777 in /usr/local/bin
With ec2-user, docker-slim command works, but my docker images which are built are with "root", so trying with root, it doesn't work.

If i go to /usr/local/bin and then do ./docker-slim, it works.

I think instructions needs to be imporved.

**Comment by kcq** [comment]
There's a couple of things to explore here @dheeruymv and i'd start with checking your PATH variable with the root user. What do you see when you do `echo $PATH` when you run as root?

**Comment by Omkar0114** [comment]
Hey @kcq, 
Same here I installed docker-slim on the system, and It's showing missing docker-connection info!
![image](https://github.com/slimtoolkit/slim/assets/88308267/39d707b7-225e-48af-83e8-83df62fe9022)

**Reply by Omkar0114** [reply]
resolved. I tried to reload and install some dependencies service and it worked. Thank You :)

**Reply by kcq** [reply]
The most common reason for the `missing Docker connection info` is the environment setup where the default Docker socket is not available. This happens with Docker Desktop because it doesn't create the default Docker socket by default. This also happens with a lot of CI/CD SaaS services where they have a custom Docker In Docker setup where they intentionally don't expose the default Docker socket.

Of course, if Docker isn't running you might get the same error :-)

Note that support for the ContainerD and Podman runtime will be available soon (no specific ETA, but it is on the near term todo list).

**Reply by Omkar0114** [reply]
Yes. You're right. For Docker Desktop we don't have `docker. sock`. Hence, when we run `docker images` we don't get it.`slim` extension images. But we get minified images when run `sudo docker images`.  
Can we add support for the docker desktop? as a feature issue?
